### ISSUE TITLE
[FR] Simplify translated function

### ISSUE THREAD

**Issue by chertvl** [PiotrMachowski/Home-Assistant-custom-components-Custom-Templates#10]
Thank you Piotr for wonderful component!

It would be great to simplify **ct_state_translated(entity,'en')** function to something like this:
```{{ localized('sun.sun') }}```
where target language is current lang of HA.
By analogy, how do we call 
```{{ states('sun.sun') }}```

Its a little hard to provide long function name "ct_state_translated" and "lang" as second object - every time.

P.s.: Of course, this should not be a breaking change, but only an additional jinja function.
Thanks!

**Comment by PiotrMachowski** [comment]
It's possible that I'll be able to add such functionality to HA core.

Prefix `ct_` was added intentionally to mark the function as provided by the custom integration and avoid potential naming conflicts in the future.

By the way, you can create a macro if you want to use this function:
```yaml
{% macro localize(entity_id) %}
{{ ct_state_translated(entity_id, 'en') }}
{% endmacro %}
```
[reusing macros](https://www.home-assistant.io/docs/configuration/templating/#reusing-templates)

**Comment by chertvl** [comment]
> By the way, you can create a macro if you want to use this function

Thanks for your reply!
This is still for 3 strings more, then 1 magic word :D 



> Prefix `ct_` was added intentionally to mark the function as provided by the custom integration and avoid potential naming conflicts in the future.

Im understand it, but using "localized" word is not neccessary for me, its just example for 1-word simplify function.
It may be something like "ctloc" or any else quite unique and simple.
{{ ctloc('sun.sun') }}  - looks comfortable too.
(with using HA language in component backend, example "fr" is my HA langause, output of function must be in french)



> It's possible that I'll be able to add such functionality to HA core.

If its possible, it would be great. If not - we will use that old massive function.

**Comment by chertvl** [comment]
Closed because of Core 2024.3
Big thanks to @PiotrMachowski !

**Comment by PiotrMachowski** [comment]
❤️
